diff --git a/integration-tests/Setup/android/build.gradle b/integration-tests/Setup/android/build.gradle
index 4c196b0..856b221 100644
--- a/integration-tests/Setup/android/build.gradle
+++ b/integration-tests/Setup/android/build.gradle
@@ -29,6 +29,7 @@ allprojects {
             // All of React Native (JS, Obj-C sources, Android binaries) is installed from npm
             url "$rootDir/../node_modules/react-native/android"
         }
+        maven { url 'https://jitpack.io' }
     }
 }
 
