From d57bb2e3a636130b2b09431f5ae3bcf057a6d4bc Mon Sep 17 00:00:00 2001
From: Piotr Dubiel <piotr.dubiel@polidea.com>
Date: Tue, 5 Feb 2019 23:16:05 +0100
Subject: [PATCH] minSdkVersion bump

---
 examples/Setup/android/build.gradle | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/examples/Setup/android/build.gradle b/examples/Setup/android/build.gradle
index 85d8f2f..27debd1 100644
--- a/examples/Setup/android/build.gradle
+++ b/examples/Setup/android/build.gradle
@@ -3,7 +3,7 @@
 buildscript {
     ext {
         buildToolsVersion = "27.0.3"
-        minSdkVersion = 16
+        minSdkVersion = 18
         compileSdkVersion = 27
         targetSdkVersion = 26
         supportLibVersion = "27.1.1"
-- 
2.10.0

