From 678c0d683725ea5383b19c084945390c96bc17a4 Mon Sep 17 00:00:00 2001
From: Piotr Dubiel <piotr.dubiel@polidea.com>
Date: Wed, 6 Feb 2019 00:07:24 +0100
Subject: [PATCH] minSdkVersion bump

---
 examples/Setup/android/build.gradle | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/examples/Setup/android/build.gradle b/examples/Setup/android/build.gradle
index 49569e4..d0e682a 100644
--- a/examples/Setup/android/build.gradle
+++ b/examples/Setup/android/build.gradle
@@ -33,7 +33,7 @@ allprojects {
 
 ext {
     buildToolsVersion = "26.0.3"
-    minSdkVersion = 16
+    minSdkVersion = 18
     compileSdkVersion = 26
     targetSdkVersion = 26
     supportLibVersion = "26.1.0"
-- 
2.10.0

