From 30dd81b13f303555afaa7121a23167a2cd1e90f3 Mon Sep 17 00:00:00 2001
From: Piotr Dubiel <piotr.dubiel@polidea.com>
Date: Fri, 8 Feb 2019 19:03:37 +0100
Subject: [PATCH] Bump minSdkVersion to 18

---
 android/build.gradle | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/android/build.gradle b/android/build.gradle
index 3231b29..4c196b0 100644
--- a/android/build.gradle
+++ b/android/build.gradle
@@ -3,7 +3,7 @@
 buildscript {
     ext {
         buildToolsVersion = "28.0.2"
-        minSdkVersion = 16
+        minSdkVersion = 18
         compileSdkVersion = 28
         targetSdkVersion = 27
         supportLibVersion = "28.0.0"
-- 
2.10.0

